src/hiero_sdk_python/transaction/transfer_transaction.py

```
@@ -1,226 +1,391 @@
 """
+
 Defines TransferTransaction for transferring HBAR or tokens between accounts.
+
 """
 
-from typing import Dict, List, Optional, Tuple
+
+from typing import Dict, List, Optional, Tuple, Union
```

Ruff (0.14.10)

`typing.Dict` is deprecated, use `dict` instead

(UP035)